observations/obs_converters/AIRS/BUILD_HDF-EOS.sh

```diff
@@ -0,0 +1,270 @@
+#!/bin/sh
+# 
+# updated 4 Dec 2020
+
```

Comment:
AIRS/BUILDME is now instructions for building HDF-EOS, so I renamed it when I updated the content describing where to get the source, what was needed, etc.